src/Error/errorHandler.js

```diff
@@ -0,0 +1,9 @@
+/**
+ *
+ * @param {string} errorMSG
+ * @summary 에러 메시지를 출력하고 프로그램을 종료한다.
+ */
+const errorHandler = (errorMSG) => {
```

Comment:
```suggestion
const errorHandler = (errorMessage) => {
```
굳이 축약하지 않아도 될 것 같은데 문경님 생각은 어떠신가요??

Reply:
저는 개인적으로 msg로 축약하는 것도 선호하는 편입니다! 요건 사바사인 것 같네요 ㅎㅎ 근데 MSG -> Msg가 더 좋을 것 같아요. 축약되는 단어의 맨 앞 문자만 대문자 처리를 해줘야할 것 같습니다!

Reply:
확실히 축약하지 않는게 전달성이 좋아보이긴 합니다.
전부 대문자로 사용한 이유는 약어의 경우 전부 대문자, 혹은 소문자를 사용하라는 [airbnb 컨벤션](https://github.com/airbnb/javascript?tab=readme-ov-file#naming--Acronyms-and-Initialisms)을 따른거긴 한데 MSG도 약어에 해당되는걸까요 ㅋㅋㅋㅋ 

`errorMessage`로 작성하는게 조금 더 좋아보이긴 하네요 :) 좋은 추천 감사합니다